packages/clay-css/src/scss/_components.scss

```diff
@@ -16,6 +16,7 @@
 @import 'components/_toasts';
 
 @import 'components/_icons';
+@import 'components/_icon-selector';
```

Comment:
@pat270 were should this be placed in this list and why? Can you give a short explanation of the logic of the order of this file?